run-local.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo "REMINDER: Run 'ng build --watch -c local' before running this"
+
+docker stop mvn-search && docker rm mvn-search
+docker build -f local.Dockerfile -t mvn-search .
+docker run \
+-p 80:80 \
+--name mvn-search \
+-v `pwd`/dist:/usr/share/nginx/html \
```

Comment:
You should be able to do: `-v $(pwd)/dist:/usr/share/nginx/html \` here

As well, I believe `docker-compose` will allow a relative path for a volume, so you might check that out (as you can see all these values using that)

Reply:
I don't believe you can do rel path with volumes. Found https://github.com/moby/moby/issues/4830 but looks like they said nope.